### DISCUSSION TITLE
do webhooks currently work?

### DISCUSSION THREAD

**Discussion by dietrichmd** [mealie-recipes/mealie#295] [Q&A]
Try as I might, I have yet to get a webhook to work on my (docker) install.  Has anyone gotten it to work?  Currently trying with discord.

**Comment by hay-kot** [comment]
> Try as I might, I have yet to get a webhook to work on my (docker) install.  Has anyone gotten it to work?  Currently trying with discord.

I've got them working with Home Assistant. I'm not sure how discord works, but you get JSON payload of the current recipe scheduled that day. Note that if there isn't one you won't get a payload, even when testing webhooks

**Comment by michael-genson** [comment]
I'm currently going through some old discussions, so apologies for the necro.

Webhooks have been stable for quite some time; if you're having issues with them, feel free to open a bug report.
